README.md

```diff
@@ -14,6 +14,13 @@ Argo Helm is a collection of **community maintained** charts for [https://argopr
 helm repo add argo https://argoproj.github.io/argo-helm
 ```
 
+## Version Support Policy
+As our project is maintained by a small team, we must focus our limited resources on following upstream projects and ensuring the stability of the latest version.
+
+Consequently, **we do not provide bug fixes or security patches for older versions.** Our official support is limited to **the latest version of the upstream projects** only.
+
+We strongly encourage all users to upgrade to the latest version to benefit from the most recent features, bug fixes, and security patches.
```

Comment:
Maybe just a sentence at the end calling for others to join the team, link to the argoproj membership process? 

Other than that, I think this is good.

Reply:
I added a commit for them. 🙏 
https://github.com/argoproj/argo-helm/pull/3466/commits/cbb122aefc2fbe49d3d072424d798358b837d984